plugin.rb

```diff
@@ -109,7 +109,7 @@ def reset_voting_cache
   class ::User
       def vote_count
         if self.custom_fields["votes"]
-          user_votes = self.custom_fields["votes"].reject { |v| v.nil? }.length
+          user_votes = self.custom_fields["votes"].reject { |v| v.empty? }.length
```

Comment:
oh, empty is not going to work on nil... so I guess what you what here is `v.blank?` which is very cross compatible, also why do we have an empty string here ... hmmm ....

Reply:
Ugh. I meant to put `blank`. I'll fix. Why there is an empty string, I have no idea though. Maybe it's because of the merged users?